wordpress-mobile/WordPress-Android
# Feature/social login analytics

Refs #6505
Adds social login analytics. These are the same as Calypso except for the following:

    "login_social_2fa_needed"
    "login_social_accounts_need_connecting"
    "login_social_error_unknown_user"

To test:
Test social login.
Ensure that each analytic is being bumped correctly.

(Companion iOS version https://github.com/wordpress-mobile/WordPress-iOS/pull/8028)
